base/include/antioch/base/config/config.h

```diff
@@ -0,0 +1,30 @@
+// Copyright Antioch. All rights reserved.
+
+#pragma once
+
+#include <antioch/transit_base/station.h>
+
+#include <nlohmann/json.hpp>
+#include <vector>
+
+#include "../types.h"
+
+namespace antioch {
+namespace base {
+
+class Config {
+ public:
+  static void validateOrThrow(const Config* config);
```

Comment:
What case convention are you using for naming your functions?

Reply:
oh ur right oops